cmd/podman/shared/create_cli.go

```diff
@@ -76,6 +77,17 @@ func addWarning(warnings []string, msg string) []string {
 
 func verifyContainerResources(config *cc.CreateConfig, update bool) ([]string, error) {
 	warnings := []string{}
+
+	var st syscall.Statfs_t
+	if err := syscall.Statfs("/sys/fs/cgroup", &st); err == nil {
```

Comment:
Can we either do this once and stick it in a bool we can read, or make a helper for the test?

Reply:
yes, good idea.  Done in the new version